satellite/console/consoleweb/server.go

```diff
@@ -249,8 +252,7 @@ func (server *Server) bucketUsageReportHandler(w http.ResponseWriter, r *http.Re
 	}
 
 	if err = server.templates.usageReport.Execute(w, bucketRollups); err != nil {
-		server.log.Error("satellite/console/server: usage report template could not be executed", zap.Error(err))
-		server.serveError(w, r, http.StatusNotFound)
+		err = errs.New("usage report template could not be executed: %v", err)
```

Comment:
We should log this error and return `StatusInternalServerError`

Reply:
it is logged in defer

Reply:
`server.templates.usageReport.Execute` writes the header already so the status cannot be changed.

Reply:
Oh, I see.
It's very confusing with the `StatusNotFound`.
Can you add a `TODO` in the `defer` mentioning what the correct status code should be?

Reply:
`template.Execute` accepts `io.Writer`, so how can it modify the header of the request? Or you are talking about smth else?

Reply:
See documentation in https://golang.org/pkg/net/http/#ResponseWriter

Reply:
You cannot if you pass the response writer, which is good for no buffering.

Mostly, the only thing that we can do here is logging the error and assigned to `nil`.